README.md

```diff
@@ -1,2 +1,50 @@
-# Microverse_capstone-1
-Capstone project from module-1
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Capstone project from module-1
+
+## Built With
+
+- Major languages : HTML , CSS & JavaScript
+- Frameworks : none
+
+## Getting Started
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Deployment
+
+[Live Demo Link]()
+
```

Comment:
Please deploy your project using git hub pages and add the live demo link to your README file